source/code-type/builders.txt

```diff
@@ -68,22 +68,22 @@ To analyze your builder expressions in Visual Studio, perform the following acti
 Simple Builder Expressions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Click the following tabs to see a builder expression with and without an
-information message displayed:
+Click on the following corresponding tab to see a builder expression
```

Comment:
Definitely not required as part of this backport, but per [style guide](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#c), use "click" instead of "click on"